runtime/dart_init.cc

```diff
@@ -124,6 +124,11 @@ static ServiceIsolateHook g_service_isolate_hook = nullptr;
 static RegisterNativeServiceProtocolExtensionHook
     g_register_native_service_protocol_extensions_hook = nullptr;
 
+// Kernel representation of core dart libraries(loaded from platform.dill).
+static void* kernel_platform = nullptr;
```

Comment:
This value once set will not be deleted anymore? I don't see an engine teardown path for us to delete this, maybe something to think about for another CL (We could also call Dart_Cleanup there which I noticed was not being called anywhere).

Reply:
Added TODO.